### ISSUE TITLE
Clarify warning about mutable properties with setters

### ISSUE THREAD

**Issue by lptr** [gradle/gradle#9552]
@ldaley thinks this message should be better:

> Warning: Type ‘com.gradle.dotcom.build.cloudflare.CreateCloudflareDnsRecordTask’: property ‘content’ with mutable type ‘org.gradle.api.provider.Property’ is redundant. Use methods on the property value itself to mutate it

The property is redundant? I think this is telling me to make the property read only, but it is certainly not clear.

**Comment by ldaley** [comment]
How about …

```
Warning: Type ‘com.gradle.dotcom.build.cloudflare.CreateCloudflareDnsRecordTask’: property ‘content’ of mutable type ‘org.gradle.api.provider.Property’ is writeable. Properties of this type should be read-only and mutated via the value itself.
```

?

It's a bit wordy, but a little bit clearer on what to do.

**Comment by paplorinc** [comment]
Fixed on 5.5, thanks
